src/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ set(WOOF_SOURCES
     g_game.c               g_game.h
     g_input.c              g_input.h
     g_nextweapon.c         g_nextweapon.h
+    g_umapinfo.c           g_umapinfo.h
```

Comment:
Please don't forget about README.md

Reply:
We don't mention `u_mapinfo.*` in the readme. Maybe we can change the copyright since this file is almost completely rewritten.